Input/Cursor.cs

```diff
@@ -167,7 +171,9 @@ public override void Draw(GameTime gameTime)
 
         Texture2D texture = Textures[TextureIndex];
 
-        Renderer.DrawTexture(texture,
-            new Rectangle(DrawnLocation.X, DrawnLocation.Y, texture.Width, texture.Height), RemapColor);
+        Renderer.DrawTexture(
+            texture,
```

Comment:
Unless the overall line length gets inconveniently wide, keep the first parameter on the same line with the function call. Especially if it's followed by a wide line.